README.md

```diff
@@ -44,11 +44,24 @@ If you need to add features to the API, the best is to [`npm link`](https://docs
 Openfisca
 ---------
 
-```sh
+Openfisca relies on Python 2. You should [install it in a `virtualenv`](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv):
+
+```bash
 cd mes-aides-ui
-npm run install-openfisca # ou pip install --upgrade -r openfisca/requirements.txt si vous utilisez un environnement virtuel
+virtualenv  --python=python2 .venv # To create your virtualenv in ./.venv (a hidden folder)
+source .venv/bin/activate # To activate your virtualenv
+pip install pip --upgrade # To make sure you're using pip latest version
+```
+
+`virtualenv  --python=python2 .venv` will fail if your python executable is not `python2`. You can try `virtualenv  --python=python2.7 .venv` and if that one fails too, you can open an issue to get some support.
```

Comment:
Do we really want to suggest opening an issue? Where should that issue be opened?